libs/net/dal/Config/S3Options.cs

```diff
@@ -0,0 +1,86 @@
+using Amazon.Runtime;
+using Amazon.S3;
+using System.ComponentModel.DataAnnotations;
+
+namespace TNO.DAL.Config;
+
+/// <summary>
+/// S3 config
+/// </summary>
+public class S3Options
+{
+    public string BucketName => Environment.GetEnvironmentVariable("S3_BUCKET_NAME") ?? "";
```

Comment:
This isn't how configuration options should be set.  This will result in always using the environment variable value `S3_BUCKET_NAME`, where it should be following the dependency injection configuration you set below `S3.BucketName`.  `Expression-bodied member` properties are not useful for configuration options.

You can review the existing code to see how all this works.  Also the following are links to information to understand it.

https://learn.microsoft.com/en-us/dotnet/core/extensions/dependency-injection
https://learn.microsoft.com/en-us/dotnet/core/extensions/configuration